pshtt/pshtt.py

```diff
@@ -748,9 +742,21 @@ def https_check(endpoint):
                 if(PT_INT_CA_FILE is not None):
                     try:
                         cert_plugin_result = None
-                        command = sslyze.plugins.certificate_info_plugin.CertificateInfoScanCommand(ca_file=PT_INT_CA_FILE)
-                        cert_plugin_result = scanner.run_scan_command(server_info, command)
-                        if(cert_plugin_result.verified_certificate_chain is not None):
+                        scanner = Scanner()
+                        command = ScanCommand.CERTIFICATE_INFO
+                        command_extra_args = {
+                            ScanCommand.CERTIFICATE_INFO: CertificateInfoExtraArguments(custom_ca_file=Path(PT_INT_CA_FILE))
```

Comment:
Same comments as previously.

```suggestion
                            command: CertificateInfoExtraArguments(custom_ca_file=Path(PT_INT_CA_FILE))
```